sktime_dl/deeplearning/mlp/_regressor.py

```diff
@@ -0,0 +1,121 @@
+__author__ = "James Large, Withington"
+
+import keras
+import numpy as np
+import pandas as pd
+
+from sklearn.base import RegressorMixin
+
+from sktime.utils.validation.supervised import validate_X, validate_X_y
+
+from sktime_dl.deeplearning.base.estimators import BaseDeepRegressor
+from sktime_dl.deeplearning.mlp._base import MLPNetwork
+
+
+class MLPRegressor(BaseDeepRegressor, RegressorMixin, MLPNetwork):
```

Comment:
Doesn't the `BaseDeepRegressor` already inherit from the `RegressorMixin`?

Reply:
Removed and committed, thanks.